packages/react-next/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx

```diff
@@ -56,103 +57,82 @@ const getColorPickerGridCellButtonClassNames = memoizeFunction(
   },
 );
 
+// Validate if the cell's color is white or not to apply whiteCell style
```

Comment:
Please keep doc comment formatting in places where it was present before.
```suggestion
/** Validate if the cell's color is white or not to apply whiteCell style */
```